docs/ref/class-based-views/flattened-index.txt

```diff
@@ -171,6 +171,7 @@ FormView
 * :meth:`~django.views.generic.base.ContextMixin.get_context_data`
```

Comment:
I don't see any methods listed more than once, so I think these `get_context_data` lines should be removed.